assets/js/modules/reader-revenue-manager/components/PublicationApprovedOverlayNotification.test.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * PublicationApprovedOverlayNotification component tests.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { createTestRegistry } from '../../../../../tests/js/utils';
+import { render } from '../../../../../tests/js/test-utils';
+import PublicationApprovedOverlayNotification, {
+	RRM_PUBLICATION_APPROVED_OVERLAY_NOTIFICATION,
+} from './PublicationApprovedOverlayNotification';
+import { CORE_UI } from '../../../googlesitekit/datastore/ui/constants';
+import {
+	VIEW_CONTEXT_MAIN_DASHBOARD,
+	VIEW_CONTEXT_MAIN_DASHBOARD_VIEW_ONLY,
+} from '../../../googlesitekit/constants';
+import { Provider as ViewContextProvider } from '../../../components/Root/ViewContextContext';
+import { UI_KEY_READER_REVENUE_MANAGER_SHOW_PUBLICATION_APPROVED_NOTIFICATION } from '../datastore/constants';
+import { CORE_USER } from '../../../googlesitekit/datastore/user/constants';
+
+describe( 'PublicationApprovedOverlayNotification', () => {
+	let registry;
+
+	beforeAll( () => {
+		registry = createTestRegistry();
+	} );
```

Comment:
We usually reset the registry after each test case (i.e. in `beforeEach()`). Is there a reason we're doing it here once?